modules/apps/frontend-js/frontend-js-item-selector-web/src/main/resources/META-INF/resources/item_selector/openItemSelectorModal.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {render} from '@liferay/frontend-js-react-web';
+
+import DetachedItemSelectorModal, {
+	TDetachedItemSelectorModal,
+} from './DetachedItemSelectorModal';
+
+/**
+ * This utility enables item selector to be rendered from any JS context.
+ */
+
+export default function openItemSelectorModal(
+	props: TDetachedItemSelectorModal<any>
+) {
+
+	// Mount in detached node; Clay will take care of appending to `document.body`.
+	// See: https://github.com/liferay/clay/blob/master/packages/clay-shared/src/Portal.tsx
+
+	return render(
+
+		// @ts-ignore
+
+		DetachedItemSelectorModal,
+		props,
+		document.createElement('div')
+	);
```

Comment:
This is a copy of existing mechanism from [openModal utility](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/modal/commands/openModal.ts).

I considered applying `openModal` itself, instead of adding a new utility, but that would require too many changes to inner component implementation. I want to keep existing implementation as intact as possible.